Build/Common.Build.CSharp.settings

```diff
@@ -84,6 +84,7 @@
     <SignManifests>false</SignManifests>
     <AssemblyOriginatorKeyFile>$(TargetsPath)\FinalPublicKey.snk</AssemblyOriginatorKeyFile>
     <DelaySign>true</DelaySign>
+    <Features>$(Features);flow-analysis</Features> 
```

Comment:
this is my workaround after reading this post https://github.com/dotnet/roslyn-analyzers/issues/2244. I think another solution would be to have all projects include Microsoft.CodeAnalysis.FxCopAnalyzers directly as a nuget package because that package will add this build property automatically.

In our case the build pipeline task is the one who is grabbing Microsoft.CodeAnalysis.FxCopAnalyzers so we don't get the benefit of the property being added to our projects.